setup.py

```diff
@@ -199,13 +199,22 @@ def build_extensions(self):
     version=version,
     author="Claudio Freire",
     author_email="klaussfreire@gmail.com",
-    classifiers=["GNU General Public License v3 :: License :: OSI Approved :: GNU General Public License v3 (GPLv3)"],
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+        "Programming Language :: Python",
+        "Programming Language :: Cython",
+        "Programming Language :: Python :: 2",
```

Comment:
This also works with python 2, so we should add that classifier

Reply:
I wasn't sure, but all the better :)